src/modeler/AutoRest.Swagger.Tests/SwaggerModelerValidationTests.cs

```diff
@@ -626,7 +626,21 @@ public void LocationPropertyWithIncorrectXmsMutability()
             var messages = GetValidationMessagesForRule<LocationMustHaveXmsMutability>("location-with-incorrect-xms-mutability.json");
             Assert.Equal(messages.Count(), 1);
         }
-    }
+
+        [Fact]
+        public void LongRunningHasExtensionValidation()
+        {
+            var messages = GetValidationMessagesForRule<LongRunningOperationsWithLongRunningExtension>("long-running-operation-without-extension.json");
+            Assert.Equal(messages.Count(), 1);
+        }
+
+        [Fact]
+        public void LongRunningHasExtensionTrueValidation()
+        {
+            var messages = GetValidationMessagesForRule<LongRunningOperationsWithLongRunningExtension>("long-running-operation-with-extension-false.json");
```

Comment:
Curious: When someone explicitly says `"x-ms-long-running-operation": false`, do we still want to surface this warning message?